src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length--31">
+      <div class="calendar__day calendar__day--date--1 calendar__day--start-day--sun"></div>
```

Comment:
```suggestion
    <div class="calendar calendar--month-length--31 calendar--start-day--sun">
      <div class="calendar__day"></div>
```
Both start-day and month-length modifiers should be added for calendar
![image](https://user-images.githubusercontent.com/96409563/212145627-fde92012-6c79-41f9-a841-03d2e04d82af.png)